ibis/backends/dask/tests/execution/test_functions.py

```diff
@@ -146,7 +146,7 @@ def test_round_decimal_with_negative_places(t, df):
 
 @pytest.mark.xfail(
     raises=OperationNotDefinedError,
-    reason="TODO - arrays - #2553"
+    reason="TODO - arrays - #2553",
```

Comment:
This looks like an improvement over Black. It seems like the following comment might have confused Black, but without the comment you should have no trailing comma.